documentation/src/main/asciidoc/topics/marshalling.adoc

```diff
@@ -141,11 +178,69 @@ include::code_examples/CustomMarshaller.java[]
 include::config_examples/custom_marshaller.xml[]
 ----
 
-
 TIP: It's possible custom marshaller implementations to access a configured white list via the
 link:{javadocroot}/org/infinispan/commons/marshall/Marshaller.html#initialize(org.infinispan.commons.configuration.ClassWhiteList)
 method, which is called during startup.
 
+== Adding Java Classes to Deserialization White Lists
```

Comment:
This only really applies to `JavaSerializationMarshaller` and `JBossUserMarshaller`,  or potentially any other custom marshaller implementation that wants to use some form of Java Serialization and utilises our internal `ClassWhiteList`.

It could be worth explicitly adding a reference to this section in the `JavaSerializationMarshaller` and `JBoss Marshalling` sections.

Reply:
thanks @ryanemerson I've added references to those sections